plugins/corona.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "covid19_data_ruby"
```

Comment:
I'm guessing this `require` is no longer needed?

Reply:
correct, missed that. it turned out the 'gem' was just a hand full of lines not adding anything compared to getting it directly from the api